commands/describe_test.go

```diff
@@ -26,3 +33,52 @@ func Test_getFunctionURLs(t *testing.T) {
 		})
 	}
 }
+
+func TestDescribeOuput(t *testing.T) {
+	spaces := regexp.MustCompile(`[ ]{2,}`)
+	cases := []struct {
+		name           string
+		function       schema.FunctionDescription
+		verbose        bool
+		expectedOutput string
+	}{
+		{
+			name: "minimal output, non-verbose",
+			function: schema.FunctionDescription{
+				FunctionStatus: types.FunctionStatus{
+					Name:        "figlet",
+					Image:       "openfaas/figlet:latest",
+					Labels:      &map[string]string{},
+					Annotations: &map[string]string{},
+				},
+				Status: "Ready",
+			},
+			verbose:        false,
+			expectedOutput: "Name:\tfiglet\nStatus:\tReady\nReplicas:\t0\nAvailable Replicas: 0\nInvocations:\t0\nImage:\topenfaas/figlet:latest\nFunction Process:\t<default>\n",
```

Comment:
@alexellis what do you think of this? these two test cases demonstrate the kind of minimal and maximal cases

Reply:
Nicely done. The only thing I might suggest is making this a block quote to make it easier to read, but there are pros and cons to that too. We can merge as is, thanks for adding the tests.